x-pack/test_serverless/api_integration/test_suites/common/index_management/settings.ts

```diff
@@ -96,27 +95,27 @@ export default function ({ getService }: FtrProviderContext) {
       // Make sure none of the settings have been removed from ES API
       expectedSettings.forEach((setting) => {
         try {
-          expect(body.defaults.index.hasOwnProperty(setting)).to.be(true);
+          expect(body.defaults.index.hasOwnProperty(setting)).to.eql(true);
         } catch {
           throw new Error(`Expected setting "${setting}" not found.`);
         }
       });
     });
 
-    it('should update an index settings', async () => {
-      const index = await createIndex();
+    xit('should update an index settings', async () => {
```

Comment:
Same as my comment above about `xit`. Also, the whole suite is skipped anyway, so might not be needed anymore?

Reply:
https://github.com/elastic/kibana/pull/184105/commits/ae3caed5ea1578ec7e73e7b92a509b905d31e34a